service_container/configurators.rst

```diff
@@ -127,12 +127,12 @@ You can configure the service configurator using the ``configurator`` option:
             app.newsletter_manager:
                 class:        AppBundle\Mail\NewsletterManager
                 arguments:    ['@mailer']
-                configurator: ['@app.email_configurator', configure]
+                configurator: 'app.email_configurator:configure'
```

Comment:
@xabbuh that's not true. 2 colons are used when doing a callable referencing a static method (the PHP standard notation ``Class::staticMethod``). The shortcut syntax for service method callables uses a single colon (i.e. something which cannot appear in a valid PHP callable), as done already for the routing configuration.